code/modules/projectiles/gun_attachables.dm

```diff
@@ -1450,10 +1452,9 @@ inaccurate. Don't worry if force is ever negative, it won't runtime.
 	var/distance = 0
 	var/turf/prev_T
 	playsound(user, 'sound/weapons/guns/fire/flamethrower2.ogg', 50, 1)
-	var/fire_delay = attachment_firing_delay
 	if(!user.skills.getRating("firearms")) //no training in any firearms
-		fire_delay += 0.3 SECONDS //untrained humans fire more slowly.
-	COOLDOWN_START(src, last_fired, fire_delay)
```

Comment:
This breaks firing delay I have no idea why you changed this

Reply:
Because you told me to not define a fire delay var and just use attachment_firing_delay. It works perfectly fine in testing for both UGL and UBF and I'm just doing what you asked me to do.

Reply:
Gone, reduced to atoms.